### PR TITLE
Build/pipfile

### PR DESCRIPTION
**Pull request by Darkborderman**
## Types of changes
- **New feature**

## Description
Add package management tool.

## Related Issue
#5

### CODE REVIEW

**Comment by Darkborderman** [comment]
@jordanSu Would you like to help me to review or assign others to review this pull request?

**Comment by jordanSu** [comment]
@Darkborderman Yap sure, I will take a look and also assign one other member to review.

**Comment by Darkborderman** [comment]
Thanks! I'll start working on next issue once this PR has been merged.

**Review by josix — APPROVED** [review]
LGTM!

File: sample.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+USED_FILE=
+STAFF_ROLE_NAME=
+SPEAKER_ROLE_NAME=
+ATTENDEE_ROLE_NAME=
+# ep-bot.py setup
+REG_CHANNEL_ID=
+LOG_CHANNEL_ID=
+ONLY_RESPOND_REG=
+DATA_PATH=
+REG_BOT_SECRET=
```
Comment (jordanSu) [inline sample.env]:
```suggestion
```
Hello @Darkborderman,
I found you include the setup for ep-bot.py
And that remind me we actually haven't used it for a long time (maybe from the very beginning we didn't use it lol)
So I suggest not include setup for it, also I believe we can even delete `ep-bot.py` in this PR (or in another PR is also fine).

**Review by jordanSu — CHANGES_REQUESTED** [review]

File: sample.env (hunk 1 above)
Comment (Darkborderman) [reply sample.env]: Sure, I'll remove `ep-bot.py` in another PR.  This PR will focus on `Pipfile` only.

Comment (Darkborderman) [reply sample.env]: Resolved.

**Review by jordanSu — APPROVED** [review]

**Comment by jordanSu** [comment]
Thank you @Darkborderman !
It looks good now so I approved and merged.

**Pull request merged**